Shaders/Monochrome.fx

```diff
@@ -5,20 +5,58 @@
  * Monochrome removes color and makes everything black and white.
  */
 
-uniform float3 Coefficients <
+#include "ReShade.fxh"
+
+uniform int Monochrome_preset <
+	ui_type = "combo";
+	ui_label = "Preset";
+	ui_tooltip = "Choose a preset";
+	//ui_category = "";
+	ui_items = "Custom\0Monitor or modern TV\0Equal weight\0Agfa 200X\0Agfapan 25\0Agfapan 100\0Agfapan 400\0Ilford Delta 100\0Ilford Delta 400\0Ilford Delta 400 Pro & 3200\0Ilford FP4\0Ilford HP5\0Ilford Pan F\0Ilford SFX\0Ilford XP2 Super\0Kodak Tmax 100\0Kodak Tmax 400\0Kodak Tri-X\0";
+> = 0;
+
+uniform float3 Custom_Coefficients <
```

Comment:
I wouldn't rename this variable, since that breaks existing presets. The display label can be changed of course.

Reply:
Settings reverted to their SweetFX names. This may break a few Reshade 3.x settings though Monochrome was never the most popular effect. On the other hand it reenables indirect compatibility with the SweetFX presets in the SweetFX settings database.

IMO however ported my code should never have messed with the naming of the settings in the first place.